src/vehicle.cpp

```diff
@@ -4620,7 +4622,9 @@ int vehicle::total_solar_epower_w() const
 
 int vehicle::total_wind_epower_w() const
 {
-    const oter_id &cur_om_ter = overmap_buffer.ter( ms_to_omt_copy( g->m.getabs( global_pos3() ) ) );
+    // TODO: fix point types
+    const oter_id &cur_om_ter =
+        overmap_buffer.ter( tripoint_abs_omt( ms_to_omt_copy( g->m.getabs( global_pos3() ) ) ) );
```

Comment:
not in the PR, but we also need a single function call that takes a local or absolute map square and returns the tripoint_abs_omt.